src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/blacklist

```diff
@@ -20,9 +20,12 @@ staticMethod hudson.model.User get java.lang.String boolean java.util.Map
 staticMethod hudson.model.User getAll
 
 # Raw file operations could be used to compromise the Jenkins master.
+staticMethod java.io.File createTempFile java.lang.String java.lang.String
+staticMethod java.io.File createTempFile java.lang.String java.lang.String java.io.File
 new java.io.File java.lang.String
 new java.io.File java.lang.String java.lang.String
 new java.io.File java.net.URI
+staticMethod java.io.File listRoots
```

Comment:
Not per se, nor indeed with the `File` constructors. My tactic is to prevent `File`s from being created in the first place, so we do not need to exhaustively blacklist all the things you might do with them. And since there is no legitimate reason for working with `File`s in a sandbox, I stand by this.